opta/core/azure.py

```diff
@@ -170,3 +171,6 @@ def bucket_exists(self, bucket_name: str) -> bool:
             return True
         except ResourceNotFoundError:
             return False
+
+    def get_all_remote_configs(self) -> Dict[str, Dict[str, "StructuredConfig"]]:
+        raise UserErrors("Feature Unsupported for Azure")
```

Comment:
instead of changing local and azure, it's easier to just raise the exception in the abstract method
like what is done here https://github.com/run-x/opta/blob/3960cd526ce82299dfeab42d366d7b2da48cbd7d/opta/core/cloud_client.py#L34

Reply:
Raising a user error is much better, because in hierarchy it is handled with a Slack support message, otherwise we'll have to duplicate that part of code again.

Reply:
We can define a new UserError (we have a couple already), but specific to no Azure support

Reply:
@nsarupr the config command already raises a user error for azure https://github.com/run-x/opta/blob/feat/nilesh/runx-1121-opta-get-service/opta/commands/config.py#L35-L36

Reply:
this would work too raise UserError("This method is not supported by this cloud provider")